src/clr-core/modal/modal-actions.element.scss

```diff
@@ -0,0 +1 @@
+@import './../styles/mixins/utils';
```

Comment:
How are we doing the layout in the modal-footer/actions area?

Reply:
The decision discussed earlier was projected content from an API point we avoid managing layouts to give them more flexibility. So bother actions and header dont need any styles so I think both the `scss` files could be removed.

There is a outline about it here
https://github.com/vmware/clarity/blob/master/docs/CODING_GUIDELINES_CORE.md#web-component-conventions

Reply:
I think this may apply to alert subcomponents as well. Should I create a separate ticket for alerts?

Reply:
Yeah anything we find related to other components we should make a separate ticket I think. That way someone can go in and evaluate all of the components at once without slowing down an open PR.

Reply:
I'm doing an alerts refactor to use layouts after i get the PR up for buttons, tags, and badges. i can take care of this if you wouldn't mind creating a github issue and assigning it to me...

Reply:
Filed #4484 and assigned to @mathisscott.